charmcraft.yaml

```diff
@@ -140,6 +140,10 @@ config:
         This value should be equal to or shorter than half the root-ca-validity.
         Changing this value will trigger generation of a new CA certificate,
         revoking all previously issued certificates.
+    certificate-number-limit:
+      type: int
+      description: >
+        Maximum number of certificates that can be issued to a single requirer.
```

Comment:
Should we have a reasonable default value?

Reply:
I couldn't think of a "reasonable" number. But since the implementation won't completely ignore the requests and will get back to them on the next hook we can have a number.
Any suggestions? 
I can imagine something close to a reasonable number of units an application can have, or a reasonable number of relations.

Reply:
My point is that we are adding this in the context of ssdlc but if we don't set a default value, I suspect nobody will use it. I think we should prescribe a "reasonable default" and users are free to bump it if needed. 

As for the value itself, 99 is reasonable? Or we could also look at the max size of the relation databag and divide it by the size of each certificate payload.

Reply:
Yeah, I'm afraid you are right, if we don't have a default no one is going to use it. I'll add one.
99 sounds good as a default, let's have something in the double digits as a starting point.